SPECS/tuned/tuned.spec

```diff
@@ -36,7 +36,7 @@ Requires: python3-configobj
 Requires: python3-dbus 
 Requires: python3-decorator
 Requires: python3-linux-procfs
-Requires: python3-perf
+Requires: (python3-perf or python3-perf-rt)
```

Comment:
@rlmenge @harshitgupta1337 have you evaluated if we could keep this as `Requires: python3-perf` and instead do a `Provides: python3-perf` from within the various kernel specs?

If that method works, it would be more sustainable than always needing to keep this `Requires: (<package> or <package2> ...)` method for each kernel package added or removed

Reply:
I agree that the method you propose would be a more sustainable method of managing this dependency of tuned.
However, if we do that - and then try installing `tuned` from a system running `kernel-rt`, how would we enforce the selection of `python3-perf-rt` (part of `kernel-rt.spec`) and not `python3-perf` (part of `kernel.spec`) ?
In the PR created by @rlmenge  (#5897 ), each kernel spec has a conflict with every other kernel spec - which could be the solution to the above.

Reply:
Hi @christopherco , I have tried the same approach that @rlmenge used in #5897 and tested it with an image build using AzureLinux toolkit with default kernel set to one of `kernel`, `kernel-rt`, and `kernel-ipe`. In each case, the `tuned` package was installed, which installed the correct type of `python3-perf` package based on the kernel type.